lib/msal-common/src/cache/CacheManager.ts

```diff
@@ -22,17 +22,20 @@ import { ThrottlingEntity } from "./entities/ThrottlingEntity";
 import { AuthToken } from "../account/AuthToken";
 import { ICrypto } from "../crypto/ICrypto";
 import { AuthorityMetadataEntity } from "./entities/AuthorityMetadataEntity";
+import { Logger } from "../logger/Logger";
 
 /**
  * Interface class which implement cache storage functions used by MSAL to perform validity checks, and store tokens.
  */
 export abstract class CacheManager implements ICacheManager {
     protected clientId: string;
     protected cryptoImpl: ICrypto;
+    protected logger: Logger;
 
-    constructor(clientId: string, cryptoImpl: ICrypto) {
+    constructor(clientId: string, cryptoImpl: ICrypto, logger: Logger) {
```

Comment:
This may be a breaking change

Reply:
Do we expect someone to use the CacheManager class from common? If not, both NodeCacheManager and BrowserCacheManager have the logger as a constructor param, just changed it so they pass it to super instead of setting it for themselves. How likely is it to be a breaking change?

Reply:
Issue isn't someone using the CacheManager directly. Issue is that browser and node use it today. So if someone is using an old version of browser/node and the newest version of common this will throw an error when the platform lib calls `new CacheManager(clientId, cryptoImpl)` and common is now expecting `new CacheManager(clientId, cryptoImpl, logger)`.

Reply:
As I'm thinking about this more, we shouldn't be passing the logger down from the platform lib to common anyway. Common should be instantiating its own logger so that the log messages properly state they are coming from common.

Reply:
That's a great point, I'll fix this. Thanks!